src/server/table.h

```diff
@@ -92,17 +89,24 @@ class LockTable {
   bool Acquire(std::string_view key, IntentLock::Mode mode);
   void Release(std::string_view key, IntentLock::Mode mode);
 
+  static uint64_t Fingerprint(std::string_view key);
```

Comment:
Consider some typedef for the fingerprint type, like `using THash = uint64_t`

Reply:
It won't grow it won't shrink, keeps less types. Std uses size_t as well after all

Reply:
giving a name to this "type" makes the code easier to read (like that it's not an index but a hash). once C++ will get distinct typedef it'll be even better, because then the compiler can guard against such errors